pkg/daemon/update.go

```diff
@@ -262,6 +239,24 @@ func (dn *Daemon) reconcilable(oldConfig, newConfig *mcfgv1.MachineConfig) *stri
 	return nil
 }
 
+func verifyUserFields(pwdUser ignv2_2types.PasswdUser) string {
+	var msg string
+	emptyUser := ignv2_2types.PasswdUser{}
+	tempUser := pwdUser
+	if tempUser.Name == "core" && len(tempUser.SSHAuthorizedKeys) > 1 {
+		tempUser.Name = ""
+		tempUser.SSHAuthorizedKeys = nil
+		if !reflect.DeepEqual(emptyUser, tempUser) {
+			msg = "Ignition passwd user section contains unsupported changes: non-sshKey changes"
+		} else {
+			glog.Info("SSH Keys reconcilable")
+		}
+	} else {
+		msg = "Ignition passwd user section contains unsupported changes: user must be core and have 1 or more sshKeys"
```

Comment:
Don't need to do in this PR, but it should be supported to entirely remove SSH keys I'd say.

Reply:
I can double check on this req after this PR and follow-on, thanks!